services/web/server/src/simcore_service_webserver/meta_db.py

```diff
@@ -0,0 +1,479 @@
+import hashlib
+import json
+import logging
+from typing import Any, Dict, List, Optional, Tuple
+from uuid import UUID
+
+import sqlalchemy as sa
+from aiopg.sa import SAConnection
+from aiopg.sa.result import RowProxy
+from pydantic.types import NonNegativeInt, PositiveInt
+from simcore_postgres_database.models.projects import projects
+from simcore_postgres_database.models.projects_version_control import (
+    projects_vc_branches,
+    projects_vc_commits,
+    projects_vc_heads,
+    projects_vc_repos,
+    projects_vc_snapshots,
+    projects_vc_tags,
+)
+from simcore_postgres_database.utils_aiopg_orm import BaseOrm
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+
+from .db_base_repository import BaseRepository
+from .meta_errors import (
+    CleanRequiredError,
+    InvalidParameterError,
+    NoCommitError,
+    NotFoundError,
+)
+from .meta_models_repos import HEAD, CommitLog, CommitProxy, RefID, SHA1Str, TagProxy
+
+log = logging.getLogger(__name__)
+
+
+def eval_checksum(workbench: Dict[str, Any]) -> SHA1Str:
```

Comment:
May I ask you to remove the eval from the name, I perceive it as a security concern. Maybe `get_checksum` or `compute_checksum`.

Reply:
security concern?
Perhaps we should start to make a list as in [here](https://melevir.medium.com/python-functions-naming-the-algorithm-74320a18278d)